data/decks/web.yaml

```diff
@@ -1,6 +1,6 @@
 ---
-name: Web test deck without HTTP Invalid Request Line
-description: This deck runs HTTP Header Field Manipulation, and the Web Connectivity test
+name: Web test deck
+description: This deck runs HTTP Header Field Manipulation and the Web Connectivity test
```

Comment:
We cache geoip lookups for up to 10 minutes, so unless a user is travelling very fast between different locations this should not create issues.